test/erlcloud_s3_test_data.hrl

```diff
@@ -0,0 +1,507 @@
+%%%-------------------------------------------------------------------
+%%% @author <Kirill Starikov kstarikov@alertlogic.com>
+%%% @copyright (C) 2016, <Alert Logic>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 15. Feb 2016 22:21
+%%%-------------------------------------------------------------------
+-author("<Kirill Starikov kstarikov@alertlogic.com>").
+
+-define(S3_BUCKET_EVENT_XML_CONFIG,
+    {xmlElement,'NotificationConfiguration','NotificationConfiguration',[],
```

Comment:
may be it really makes sense to store test example as plain XML string and not xmerl representation?